docs/articles/nunit/writing-tests/constraints/DictionaryContainsValueConstraint.md

```diff
@@ -22,12 +22,12 @@ Does.Not.ContainValue(object)
 ```csharp
 ...Using(IComparer comparer)
 ...Using(IEqualityComparer comparer)
-...UsingPropertiesComparer()  // From version 4.1
 ...Using<T>(IComparer<T> comparer)
 ...Using<T>(Comparison<T> comparer)
 ...Using<T>(Func<T, T, bool> comparer)
 ...Using<T>(IEqualityComparer<T> comparer)
-...Using<TCollectionType, TMemberType>(Func<TCollectionType, TMemberType, bool> comparison)
+...Using<TActualValue, TExpected>(Func<TActualValue, TExpected, bool> comparison)
```

Comment:
Remove the `Value` suffix as it isn't used elsewhere.

Reply:
The `Value` here comes from the fact that it's the dictionary's `TValue`. I could rename it to `TActualDictionaryValue` to make that more clear, or the non-specific `TActual` works too, which the XML comments can clarify for those curious. Let me know what's preferable.

Reply:
For the other comparers you used the suffix `Element` to indicate this is part of a collection.
So I suggest to use `TActualValueElement` to keep both